calico/reference/felix/configuration.mdx

```diff
@@ -168,7 +168,9 @@ See the [HOWTO guide](../../operations/ebpf/enabling-ebpf.mdx) for step-by step
 | BPFLogLevel / <br/> FELIX_BPFLOGLEVEL                                               | The log level used by the BPF programs. The logs are emitted to the BPF trace pipe, accessible with the command `tc exec BPF debug`.                                                                                                                                                                                                                                              | Off,Info,Debug        | Off     |
 | BPFDataIfacePattern / <br/> FELIX_BPFDATAIFACEPATTERN                               | Controls which interfaces Felix should attach BPF programs to in order to catch traffic to/from the external network. This needs to match the interfaces that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to NodePorts and services from outside the cluster. It should not match the workload interfaces (usually named cali...).. | regular expression    | ^((en&#124;wl&#124;ww&#124;sl&#124;ib) [Popsvx]&#46;&#124;&#42; &#124;(eth &#124;wlan &#124;wwan)&#46;&#124;&#42;&#124;tunl0$ &#124; vxlan.calico$ &#124; wireguard.cali$ &#124; wg-v6.cali$) |
 | BPFL3IfacePattern / <br/> FELIX_BPFL3IFACEPATTERN                                   | Allows to list tunnel devices like wireguard or vxlan (i.e., L3 devices) in addition to BPFDataIfacePattern. That is, tunnel interfaces not created by Calico, that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to nodeports and services from outside the cluster.                                                                 | regular expression    | ""      |
-| BPFConnectTimeLoadBalancingEnabled / <br/> FELIX_BPFCONNECTTIMELOADBALANCINGENABLED | Controls whether Felix installs the connect-time load balancer. In the current release, the connect-time load balancer is required for the host to reach kubernetes services.                                                                                                                                                                                                     | true,false            | true    |
+| BPFConnectTimeLoadBalancingEnabled / <br/> FELIX_BPFCONNECTTIMELOADBALANCINGENABLED | Controls whether Felix installs the connect-time load balancer. This option will be deprecated, use the combination of bpfConnectTimeLoadBalancing and bpfHostNetworkedNATWithoutCTLB.                                                                                                                                                                                            | true,false            | true    |
```

Comment:
idk, tell me which way you want it, and I will do it that way. However, it feels confusing to me to read about an option in docs and then go and verify if that option is still live. Me as a user would not do that, but I am happy to follow an established pattern, less work for me :-P

Reply:
Let's leave the description as it is. I'll put up another PR with a deprecation notice in the release notes for 3.27+. Do you know yet when this will be removed?